AsyncDisplayKit/ASPagerNode.h

```diff
@@ -120,6 +120,24 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSInteger)indexOfPageWithNode:(ASCellNode *)node;
 
+/**
+ * Tells the pager node to allow its view controller to automatically adjust its content insets.
+ *
+ * @see UIViewController.automaticallyAdjustsScrollViewInsets
+ *
+ * @discussion ASPagerNode should usually not have its content insets automatically adjusted
+ * because it scrolls horizontally, and flow layout will log errors because the pages
+ * do not fit between the top & bottom insets of the collection view.
+ *
+ * The default value is NO, which means that ASPagerNode expects that its view controller will
+ * have automaticallyAdjustsScrollViewInsets=NO.
+ *
+ * If this property is NO, but your view controller has automaticallyAdjustsScrollViewInsets=YES,
+ * the pager node will set the property to NO and log a warning message. In the future,
```

Comment:
It will leave the pager property (as NO) but it'll automatically set NO on the view controller (for backwards compatibility) and log a message that it did so. In the future, we want users to be responsible for configuring their own view controllers, so we'll just log the message.